lib/util.js

```diff
@@ -151,7 +151,8 @@ exports.debuglog = function(set) {
     debugEnviron = process.env.NODE_DEBUG || '';
   set = set.toUpperCase();
   if (!debugs[set]) {
-    if (new RegExp(`\\b${set}\\b`, 'i').test(debugEnviron)) {
+    var sets = debugEnviron.split(',').map((s) => s.toUpperCase());
```

Comment:
The code on line 151, inside the `debugEnviron === undefined` check, should only run once though, right? In the current position, it will run each time a new set is introduced.